packages/suite/src/actions/wallet/blockchainActions.ts

```diff
@@ -262,39 +271,58 @@ export const unsubscribe = (removedAccounts: Account[]) => (_: Dispatch, getStat
     return Promise.all(promises as Promise<any>[]);
 };
 
+const tryClearTimeout = (timeout?: ReturnType<typeof setTimeout>) => {
+    if (timeout) clearTimeout(timeout);
+};
+
+export const syncAccounts =
+    (symbol: Network['symbol']) => async (dispatch: Dispatch, getState: GetState) => {
+        // First clear, to cancel last planned sync
+        tryClearTimeout(getState().wallet.blockchain[symbol].syncTimeout);
+
+        await Promise.all(
+            getState()
+                .wallet.accounts.filter(a => a.symbol === symbol)
+                .map(a => dispatch(accountActions.fetchAndUpdateAccount(a))),
+        );
+
+        // Second clear, just to be sure that no other sync was planned while executing this one
```

Comment:
Not easily. There's always a possibility that e.g. sync was initiated by notification from server just before previous sync has ended, and we a) definitely want to perform that second sync as there might be new data and b) should replan/postpone the next sync because now it doesn't have to be that soon. It would be easier to remove the first clear, but it's sort of optimization not to trigger the planned sync while there is already unplanned sync running.